services/121-service/src/env.ts

```diff
@@ -0,0 +1,217 @@
+import { createEnv } from '@t3-oss/env-core';
+import { withoutLeadingSlash, withoutTrailingSlash } from 'ufo';
+import { v4 as createUuid } from 'uuid';
+import { z } from 'zod/v4';
+
+// See: https://env.t3.gg/docs/core
+export const env = createEnv({
+  // eslint-disable-next-line n/no-process-env -- We need to give access to the actual values (at leas once)
+  runtimeEnv: process.env,
+  emptyStringAsUndefined: true,
+
+  /**
+   * See explanations for each variable in `services/.env.example`
+   * This file follows the same order/structure.
+   *
+   * Guidelines:
+   * - Use as many _specific_ requirements as possible, like `.min(8)`, `.email()`, `.url()`, etc.
+   *   See, for built-in possibilities: https://zod.dev/api
+   * - Use `.optional()` if the service should be able to start-up without any value set.
+   *   This applies to non-critical features, but also for ANY FSP-related variables!
+   *   ANY FSP should be concidered "only optional", so thay each instance of the platform ONLY has to set the FSP-related values that are in use.
```

Comment:
```suggestion
   *   ANY FSP should be considered "only optional", so that each instance of the platform ONLY has to set the FSP-related values that are in use.
```

Reply:
No really. It had disabled itself. :D